data/preprocessor.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-use-before-define */
+import webLogger from '../src/app/helpers/logger.web';
+
+const Preprocessor = (jsonRaw = {}, logger = webLogger) => {
+  try {
+    // safely get deeply nested JSON values
+    const firstPublished = get(['metadata', 'firstPublished'], jsonRaw);
+    const lastUpdated = get(['metadata', 'lastUpdated'], jsonRaw);
+    const canRenderTimestamp =
```

Comment:
very very fussy change but can you add a newline between lines 8 and 9 plz. Separates the different types of variables (data and assertions). Happy to leave if you disagree, im probably being too fussy

Reply:
Choosing to ignore, as the 3 consts are related.